### PR TITLE
Move fixed hostnames to strict whitelist.

### LINKED ISSUE #2306: Fix regex in whiltelists
`bor.*.golem.network` allows domain  `borsgolem.network` what is incorrect

### PR DESCRIPTION
**Pull request by evik42**
Fix wildcards to regex.

Fixes https://github.com/golemfactory/yagna/issues/2306.

### CODE REVIEW

**Review by nieznanysprawiciel — APPROVED** [review]

File: whitelist/strict.lst [hunk 1]
```diff
@@ -2,4 +2,14 @@
...
\ No newline at end of file
+geth.golem.network
+mainnet.infura.io
+polygon-mainnet.infura.io
+polygon-rpc.com
+arb1.arbitrum.io
+api.zksync.io
+ipfs.io
+dl.dropboxusercontent.com
+docker.io
```
Comment (pwalski) [inline whitelist/strict.lst:13]:
```suggestion
docker.io
hub.docker.com
registry.hub.docker.com
```

File: whitelist/strict.lst [hunk 2]
```diff
@@ -2,4 +2,14 @@
...
+github.com
```
Comment (pwalski) [inline whitelist/strict.lst:16]:
```suggestion
github.com
api.github.com
```

File: whitelist/strict.lst (hunk 1 above)
Comment (pwalski) [reply whitelist/strict.lst:13]: Docker's REST api is accessed through path /api, but giving access to `hub.docker.com` probably makes sense

File: whitelist/strict.lst (hunk 2 above)
Comment (pwalski) [reply whitelist/strict.lst:16]: Probably we should give access to api as well

**Review by pwalski — CHANGES_REQUESTED** [review]

File: whitelist/strict.lst (hunk 1 above)
Comment (pwalski) [reply whitelist/strict.lst:13]: Actually just checked that `registry.hub.docker.com` is an old domain, and now everything can be accessed through `docker.io`

**Review by pwalski — APPROVED** [review]

**Review by pwalski — APPROVED** [review]

**Pull request merged**